lessons/10-css-selectors/index.html

```diff
@@ -0,0 +1,61 @@
+<html>
+    <head>
+        <title>Style attributes</title>
+    </head>
+    <style>
+        header{
+            text-align: center
+        }
+    p{
+        text-align: right
+    }
+    .colbckg{
+        background-color: blue;
+        color: rgb(255,255,255)
+    }
+    .firstpara{
+        font-style: italic
+    
+    }
+    .paras{
+        text-align:right
+    }
+    .bckg{
+        background: #000fff
+    }
+    .para5{
+        text-transform: uppercase
+    }
+    .para6{
+        text-transform: capitalize
+    }
+    </style>
+    <body>
+        <header>
+            <h1>HOME PAGE</h1>
+        </header>
+        <DIV>
+            <p class="firstpara">
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequatur repellat voluptatibus culpa labore ipsum iste quidem excepturi molestias eveniet quasi, cum repudiandae non totam iusto maxime adipisci. Exercitationem, officia veniam?
+            </p>
+            <p class="colbckg">
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Doloribus nemo quo ipsa perferendis veniam reprehenderit quos? Doloribus commodi consectetur atque. Quidem rerum veritatis repellat, impedit dolor magni iusto veniam ea.
+            </p>
+            <p class="thirdpara">
+                    <span style="font-size: 24px" > Lorem </span> ipsum dolor sit amet consectetur, adipisicing elit. Obcaecati, hic eaque numquam earum aspernatur quam, vitae modi sunt quaerat at amet ea officiis ipsam! Explicabo enim iste ex fuga obcaecati!
+            </p>
+        </DIV>
+        <div>
+            <p class=bckg ;paras>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Neque est sed sunt ex. Cumque quo sequi sunt tempora eveniet nesciunt accusantium esse? Provident harum culpa eaque adipisci quaerat deleniti totam!
+            </p>
+            <p class="para5 ;paras">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt fugit laborum consequatur dolorem omnis, nam assumenda nisi deserunt asperiores quo iusto dolorum cum quia unde autem vero error soluta amet!
+            </p>
+            <p class="para6 ;paras">
```

Comment:
These class names are missing quotation marks. And if we need two classes we separate them by a space, not a semi-colon

So these would become

`<p class="para6 paras">`